eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -30,6 +32,13 @@ def create_enrollment(*args, **kwargs):
     validation_errors = check_edxapp_enrollment_is_valid(*args, **kwargs)
     if validation_errors:
         return None, [", ".join(validation_errors)]
+    if email != "":
```

Comment:
This essentially gives higher precedence to the email. Do we want this? what if the api call includes both username and email, what if the username and the email point to a different user object?

Reply:
Added an error when both are send